config/locales/in_person_proofing/en.yml

```diff
@@ -6,6 +6,16 @@ en:
         info: 'If your current address does not match the address on your ID, you will
           need to bring proof of your current address to the Post Office.'
         learn_more: Learn more
+      location:
+        location_button: 'Select'
+        location_step_about: 'If you are having trouble adding your ID, you may be able
+          to verify in person at a local United States Post Office in select
+          locations.'
+        post_office: 'Post Office ™'
+        retail_hours_heading: 'Retail Hours'
+        retail_hours_sat: 'Sat: '
+        retail_hours_sun: 'Sun: '
+        retail_hours_weekday: 'Mon-Fri: '
```

Comment:
in my experience, it's easier to manage trailing spaces in templates, etc than to put them in translation files because the spaces are not always preserved across locales when they came back from vendors

Reply:
just to clarify - to manage the trailing space would it be preferable to do something like this on the frontend: 
translation string + "space" + data from the api